### PR TITLE
closes #1

### LINKED ISSUE #1: CQRS & Event sourcing
Implements CQRS and Event sourcing patterns.

### PR DESCRIPTION
**Pull request by cichan02**

### CODE REVIEW

File: src/main/java/by/piskunou/solvdlaba/service/impl/AirlineQueryServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+	@Transactional(readOnly = true)
+	public List<AirlineAggregate> findAll() {
+		return repository.findAll();
+	}
+
+	@Override
+	@Transactional(readOnly = true)
+	public AirlineAggregate findById(UUID id) {
+		return repository.findById(id)
+						 .orElseThrow(() -> new ResourceNotExistsException("There's no airline with such id"));
```
Comment (h1alexbel) [inline src/main/java/by/piskunou/solvdlaba/service/impl/AirlineQueryServiceImpl.java:30]: line is too long

File: src/main/java/by/piskunou/solvdlaba/web/controller/AdviceController.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        return new ErrorResponseDTO(e.getMessage());
+    }
+
+    @ExceptionHandler(MethodArgumentNotValidException.class)
+    @ResponseStatus(HttpStatus.BAD_REQUEST)
+    public List<ErrorResponseDTO> handleBindException(MethodArgumentNotValidException e) {
+        List<ErrorResponseDTO> responses = new ArrayList<>();
+        responses.addAll(e.getGlobalErrors()
+                          .stream()
+                          .map(globalError -> new ErrorResponseDTO(globalError.getObjectName(), globalError.getDefaultMessage()))
```
Comment (h1alexbel) [inline src/main/java/by/piskunou/solvdlaba/web/controller/AdviceController.java:37]: line is too long

File: src/main/java/by/piskunou/solvdlaba/web/dto/groups/onCreate.java [hunk 3]
```diff
@@ -0,0 +1,5 @@
+package by.piskunou.solvdlaba.web.dto.groups;
+
+public interface onCreate {
```
Comment (h1alexbel) [inline src/main/java/by/piskunou/solvdlaba/web/dto/groups/onCreate.java:3]: please follow Java code conventions

File: src/main/java/by/piskunou/solvdlaba/web/dto/groups/onSearch.java [hunk 4]
```diff
@@ -0,0 +1,5 @@
+package by.piskunou.solvdlaba.web.dto.groups;
+
+public interface onSearch {
```
Comment (h1alexbel) [inline src/main/java/by/piskunou/solvdlaba/web/dto/groups/onSearch.java:3]: please follow Java code conventions

**Review by h1alexbel — CHANGES_REQUESTED** [review]

File: src/main/resources/application.yaml [hunk 5]
```diff
@@ -4,10 +4,31 @@ spring:
...
+  liquibase:
+    enabled: true
+    change-log: classpath:db.changelog/db.changelog-master.yaml
+    default-schema: ${LIQUIBASE_SCHEMA}
+
   #Kafka
   kafka:
     bootstrap-servers: ${MAIN_HOST}:9092
 
+  elasticsearch:
```
[the remark this answers is not in the document]
Comment (cichan02) [reply src/main/resources/application.yaml:28]: I tried to use, but something got wrong 😢

**Review by h1alexbel — APPROVED** [review]

**Pull request merged**